homeassistant/components/lupusec/__init__.py

```diff
@@ -73,10 +75,50 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up this integration using YAML is not supported."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up this integration using UI."""
+
+    if hass.data.get(DOMAIN) is None:
+        hass.data.setdefault(DOMAIN, {})
+
+    host = entry.data.get(CONF_HOST)
+    username = entry.data.get(CONF_USERNAME)
+    password = entry.data.get(CONF_PASSWORD)
+    name = entry.data.get(CONF_NAME)
+
+    try:
+        lupusec_system = await hass.async_add_executor_job(
+            LupusecSystem, username, password, host, name
+        )
+        hass.data[DOMAIN] = lupusec_system
```

Comment:
Only put stuff in a try block that can raise

Reply:
removed hass.data[domain] from catch block